PowerPointLabs/PowerPointLabs/HighlightBulletsText.cs

```diff
@@ -202,8 +205,11 @@ private static List<int> GetUnselectedParagraphs(Office.TextRange2 textRange, Of
                 {
                     continue;
                 }
+
+                bool hasCarriageReturn = textRange.Paragraphs[i].Text.EndsWith("\r");
+
                 if (selectedText.Start + selectedText.Length < paragraph.Start ||
-                    selectedText.Start > paragraph.Start + paragraph.Length - 1)
+                    selectedText.Start > paragraph.Start + paragraph.Length - 1 + (hasCarriageReturn ? 0 : 1))

```

Comment:
Wouldn't it be better to assign the 0/1 to `hasCarriageReturn` (maybe use a different name)  instead?